### PR TITLE
Components

### PR DESCRIPTION
**Pull request by JuanLPalacios**
implementation of React component that will hold the core calculator functionality.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
```
Comment (DeliceLydia) [inline src/App.js]: - [x] I suggest that you change The `App.js` component to be class-based.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      <button type="button" className="orange">+</button>
+
+      <button type="button" className="w-2">0</button>
+      <button type="button">.</button>
+      <button type="button" className="orange">=</button>
+    </div>
+  );
+}
+
+export default Calculator;
```
Comment (DeliceLydia) [inline src/components/Calculator.js]: - [x] The `Calculator.js` component should be class-based as well

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @JuanLPalacios, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:
- Calculator looks exactly like the design
- Followed git workflow
- No linter errors


## Required Changes ♻️
_Check the comments under the review._
 - [x] Consider deleting each and every file you are not using like in the `public` folder you can just keep `index.html`
 - [x] In the `src` folder you don't need `reportWebVitals.js` etc

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by mikethreels — APPROVED** [review]
Hi Juan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**